package.json

```diff
@@ -24,5 +24,8 @@
   "license": "MIT",
   "devDependencies": {
     "lerna": "^4.0.0"
+  },
+  "dependencies": {
+    "react-router-dom": "^6.2.1"
```

Comment:
Keep this in the app local `package.json`

Reply:
done